README.md

```diff
@@ -23,27 +23,46 @@ you will need to have PHP installed to run it.
 Install it with your preferred package management tool
 (for example, on Ubuntu Linux you can run: `sudo apt-get install php`)
 
+The recommended way to install this library is with Composer.
+Composer is a dependency management tool for PHP
+that allows you to declare the dependencies your project needs
+and installs them into your project.
+
 Install Composer by following the instructions [here](https://getcomposer.org/doc/00-intro.md).
 
-Then, download Wikimate, and initialise it by running `composer install`
-(or `composer.bat install` if you're on Windows).
+Then, run the following command in your project's folder
+to download Wikimate and initialise it:
+
+```sh
+composer require hamstar/Wikimate
+```
+
+(or `composer.bat require hamstar/Wikimate` if you're on Windows).
 
-To use Wikimate within another project, you can add it as a composer dependency
-by adding the following to your `composer.json` file:
+To use Wikimate within another project, you can add it as a Composer dependency
+by adding the following to your existing `composer.json` file:
 
 ```json
-"hamstar/Wikimate": "0.15.0"
+{
+    "require": {
+        "hamstar/Wikimate": "^0.15.0"
```

Comment:
It doesn't matter massively before 1.0, but I'd suggest writing this as the slightly more lenient `^0.15`. This is because (once the trailing zero becomes a 1 or whatever), if a project depends on two libraries that both depend on Wikimate, there's less chance of version conflicts. This doesn't really matter before 1.0, and anyway it's probably not that likely for multiple libraries to depend on Wikimate, so feel free to ignore this comment! :-)

Reply:
> It doesn't matter massively before 1.0, but I'd suggest writing this as the slightly more lenient `^0.15`. This is because (once the trailing zero becomes a 1 or whatever), if a project depends on two libraries that both depend on Wikimate, there's less chance of version conflicts. This doesn't really matter before 1.0, and anyway it's probably not that likely for multiple libraries to depend on Wikimate, so feel free to ignore this comment! :-)

Thanks, I made the change.
But I am sort of wondering if what you wrote applies more to `~1.0` rather than `^1.0` (1.0.0 will indeed be our next release).

Per [the documentation](https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators) my understanding is that `^1.0.0` will allow ">=1.0.0 <2.0.0" and is thus very lenient, while `~1.0.0` allows only ">1.0.0 <1.1.0" and thus indeed increases the risk of a version conflict. But I'm new to this, so open to enlightenment. :)